docs/tractus-x-release.md

```diff
@@ -0,0 +1,131 @@
+
+Once per year, Tractus-X releases a major release, making the remaining three releases minor releases. A major release may contain critical breaking changes that have a major impact on data space participants, such as changes to enablement services.
+
+A minor release contains backward compatible functionality. In addition, patch versions provide backward compatible bug and security fixes.
+
+The overarching Tractus-X releases are coordinated by the release management group (SIG Release). This team collaborates with the currently released versions of Tractus-X products to formulate the overarching release.
+
+As mentioned in [planning.md](planning.md) the Tractus-X project follows an overarching qualification process that all official quarterly Tractus-X releases adhere to. This process applies E2E activities for testing and security for all relevant release participants in compound, which stakeholders can build on. Each product can/shall be released on demand on a higher cadence, see [product_release.md].
+
+
+## Scope / Content
+
+The Tractus-X release process aims to ensure compatibility of Tractus-X products and defines common quality criteria.
+
+The desired enhancement of the dataspace functionality is planned in the [open refinement and planning](open-refinement-and-planning.md) sessions.
+
+Past release documentation can be found in [eclipse-tractusx/tractus-x-release](https://github.com/eclipse-tractusx/tractus-x-release).
+
+Note: In every quarterly TX release, we shall expect evolutions (new versions) of some products (see [product_release.md]). But also some other products may NOT face a change/up-revision in the given iteration. Nevertheless, those products (“carry-over”) are still essential parts of the release bundle.
+
+Products **must** opt-in to the process in order to be included in a Tractus-X release. (see product release.md)
+
+
+## Process abstract (future state; or “to-be”)
+
+- Planning
+- Feature Work and Product Release
+- Quality assurance in parallel
+- Opt-In for participation in TX Release
+- Completion of Q-Assurance
+- TX Release
+
+Once feature work on product level is initiated, the quality assurance of the foreseen release package runs in parallel. The vehicle of Umbrella helm charts is used to continuously test pre-defined product bundles. Automatically and environment-less.
+
+Reference onion rings: (add URL)
+
+Example for cxOS (https://catena-x.net/fileadmin/_online_media_/231006_Whitepaper_Tractus-X.pdf (https://catena-x.net/fileadmin/_online_media_/231006_Whitepaper_Tractus-X.pdf):
+
+A configured umbrella helm chart starts-up all relevant products. No matter if carry-over versions of a previous TX release or evolutions of the current iteration. User Journey tests (e2e) are automatically performed - customized for the respective bundle of products and within given boundary conditions.
```

Comment:
No matter if carry-over product versions of a previous TX release or product evolutions, developed within the current iteration.

Reply:
So we reference Tractus-X now as TX? Shouldn't we just keep Tracuts-X? Its short enough

Reply:
Text adjusted to suggestion

Reply:
compromise proposal:
No matter if carry-over versions of a previous Tractus-X release or evolutions of the current iteration. User Journey tests (E2E) are automatically performed - customized for the respective bundle of products and within given boundary conditions.

Reply:
done